planning/behavior_path_planner/src/scene_module/utils/geometric_parallel_parking.cpp

```diff
@@ -482,9 +482,12 @@ PathWithLaneId GeometricParallelParking::generateArcPath(
 
   // insert the last point exactly
   const auto p = generateArcPathPoint(center, radius, end_yaw, is_left_turn, is_forward);
-  path.points.push_back(p);
+  constexpr double min_dist = 0.01;
+  if (calcDistance2d(path.points.back(), p) > min_dist) {
```

Comment:
Do `path.points.empty()` check before using the `path.points.back()`

Reply:
thaks, fixed in 7ad01a04d79d0ebe4aaec2ec77ebdfbefbbd227b